src/applications/coronavirus-vaccination/apiCalls/index.js

```diff
@@ -0,0 +1,25 @@
+import { apiRequest } from 'platform/utilities/api';
+
+export const apiGetRequest = async url => {
+  return apiRequest(url);
+};
+
+export const apiPostRequest = async (url, data) => {
+  return apiRequest(url, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
+
+export const apiPatchRequest = async (url, data) => {
+  return apiRequest(url, {
+    method: 'PATCH',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
```

Comment:
It's very similar, but I created a new directory for the API requests. You should be able to pull it off of master via 

```
git pull
git checkout origin/master src/applications/coronavirus-vaccination/api/index.js
```

`apiGetRequest` => `retrievePreviouslySubmittedForm`
`apiPostRequest` => `saveForm`

Reply:
Got it, thanks Nick!